rusk/src/lib/node/rusk.rs

```diff
@@ -683,40 +682,80 @@ fn accept(
     ))
 }
 
-fn reward_slash_and_update_root(
+/// Execute rewards and slashes in a VM session.
+///
+/// The state root is updated accordingly.
```

Comment:
This is not the sate root, but it's the transfer merkle tree note root

Reply:
Fixed